sympy/stats/error_prop.py

```diff
@@ -0,0 +1,62 @@
+"""Tools for arithmetic error propogation."""
+from __future__ import print_function, division
+from itertools import repeat
+from sympy import Symbol, symbols, Add, Mul, simplify, Pow, exp
+
+
+def variance_prop(expr, consts=()):
+    """Symbolically propagates variance (sigma^2) for expressions.
+
+    Parameters
+    ==========
+    expr : Expr
+        A sympy expression to compute the variance for.
+    consts : sequence of Symbols, optional
+        Represents symbols that are known constants in the expr,
+        and thus have zero variance.
```

Comment:
Maybe make it more explicit that all symbols that aren't in `consts` are assumed to be variant (is that the right word?).